static/src/js/components/GranuleDetails/GranuleDetailsInfo.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import './GranuleDetailsInfo.scss'
+
+export const GranuleDetailsInfo = ({ xml }) => {
+  let content = xml
+
+  content = content.replace('<Granule>\n', '') // Remove top level element
```

Comment:
If there aren't tests for this, we should add some and/or ensure that these actually work. These appear to be exact duplicates of regexes ported from ruby and its generally not a safe assumption that regexes across languages are identical.

Reply:
They're not exact duplicates. There were some changes that needed to be made. Definitely understand that there may be some inconsistencies between languages. I thought testing the transformed output would be sufficient here. If not, I'll need a little guidance as to how we would like to test them.